regression.py

```diff
@@ -25,6 +24,56 @@ def read_info():
   info.reset_index(drop=True, inplace=True)
   return info
 
+def gic_stats(data_types=['GIC']):
+  import pickle
+  temp_pkl = CONFIG['dirs']['data'] + '\_regression\gic_stats.pkl'
+
+  if os.path.exists(temp_pkl):
+    # read gic_std and gic_max from it.
+    logger.info(f"  Reading gic_std and gic_maxabs from {temp_pkl}")
+    with open(temp_pkl, 'rb') as f:
+      gic_std, gic_maxabs = pickle.load(f)
+
+  else:
+    from swerve import site_read, site_stats, read_info_df
+
+    info_df = read_info_df(exclude_errors=True)
+    #info_df = read_info()
+    sites = info_df['site_id']
```

Comment:
Do a query here for rows with GIC, measured, {NERC, TVA} so that your loop over sites is more constrained. Won't need the "if" statement in the loop then?